lib/foodcritic/command_line.rb

```diff
@@ -83,6 +107,68 @@ def initialize(args)
           e.recover args
         end
       end
+
+      [:cookbook_paths, :role_paths, :environment_paths, :include_rules, :exclude_paths].each do | pth|
+        @options[pth].map! { |c| File.expand_path(c, '.') } if @options[pth] && @options[pth] != [nil]
+        @options[pth].flatten!
+        @options[pth].uniq!
+      end
+    end
+
+    # Search for and load config files
+    def get_config(cbs)
+      require 'app_conf'
+      files_to_load = []
+      files_to_load << File.expand_path('~/.chef/foodcritic.yml', '.') if File.file?(File.expand_path('~/.chef/foodcritic.yml'))
+      ['.', cbs].flatten!.each do |mypth|
+        tmp_path = File.expand_path(mypth, '.')
+        until File.basename(tmp_path) == '/'
+          files_to_load << "#{tmp_path}/.chef/foodcritic.yml" if File.file?("#{tmp_path}/.chef/foodcritic.yml")
+          tmp_path = File.expand_path(tmp_path + '/..','.')
+        end
+      end
+      if @original_args.include?('--config')
+        ind = @original_args.index('--config') + 1
```

Comment:
Ok, I did it this way because I originally had get_config running before the OptionParser to ensure the command line took precedence.   As things got re-arranged, I never went back and re-worked this.  I'll fix this as well.